oneargopy/Argo.py

```diff
@@ -210,6 +208,7 @@ def select_profiles(self, lon_lim: list = [-180, 180], lat_lim: list = [-90, 90]
         print(f'Floats: {narrowed_profiles.keys()}')
```

Comment:
only if verbose; add \n
delete lines 209-211

Reply:
![image](https://github.com/user-attachments/assets/4cc51cec-d015-4eb5-8d4c-40c49c024090)

Is this okay?